lib/package-managers/npm.js

```diff
@@ -48,6 +48,31 @@ function parseJson(result, data) {
   return json
 }
 
+/**
+ * Check if package author changed between current and next version.
+ *
+ * @param packageName Name of the package
+ * @param currentVersion
+ * @param latestVersion
+ * @returns A promise that fullfills with boolean value.
+ */
+async function packageAuthorChanged(packageName, currentVersion, latestVersion) {
+  npmConfig.fullMetadata = true
+
+  const result = await pacote.packument(packageName, npmConfig)
+  if (result.versions) {
+    const pkgVersions = Object.keys(result.versions)
+    const current = currentVersion
+    const latest = semver.maxSatisfying(pkgVersions, latestVersion)
```

Comment:
Since `latestVersion` is not a range, I don't think `maxSatisfying` is a NOOP here.

Reply:
What do you mean? The `maxSatisfying` takes as first argument an array of the available versions and the second argument either a `Range` or a `string` specifying the version to satisfy.

Reply:
Sorry, I meant I *think* `maxSatisfying` is a NOOP.

Here is the definition of `maxSatisfying`:

> `maxSatisfying(versions, range)`: Return the highest version in the list that satisfies the range, or null if none of them do.

Since `latestVersion` is a concrete version as opposed to a range, won't `semver.maxSatisfying(pkgVersions, latestVersion)` always return `latestVersion`?

Reply:
Tested and you are right, it is the same version. Also there is a big downside: if the latest provided version doesn't exist (which I guess should never happen), it returns `null` because can't find a satisfying version.

Reply:
I think it's safe to assume `latestVersion` will always exist.